CITATION.cff

```diff
@@ -0,0 +1,20 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: unknown data framework
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Benedikt Maria
+    family-names: Beckermann
+    affiliation: Schloss Dagstuhl Leibniz-Zentrum für Informatik
```

Comment:
Please format the affiliation as `Schloss Dagstuhl - Leibniz Center for Informatics`